third_party/jit/cpu.cc

```diff
@@ -315,11 +313,31 @@ void CPU::Initialize() {
   }
 }
 
+#if defined( __linux__)
+#include <sys/sysinfo.h>
+
 int CPU::Processors() {
-  Probe();
   return get_nprocs();
 }
 
+#elif defined(__APPLE__)
```

Comment:
do we still need this and the other header guards?

Reply:
The get_nprocs(3) function is a "GNU extension" whereas the rest of the JIT assembler code is POSIX compliant as far as I can tell, so this is to avoid making the code Linux-specific. We could drop the `#ifdef __APPLE__` part, but we would need to keep the `#else` part for the code to compile on all POSIX platforms.